packages/plugin-window-onerror/test/onerror.test.js

```diff
@@ -42,8 +44,9 @@ describe('plugin: window onerror', () => {
     // it('captures uncaught errors in DOM (level 2) event handlers', done => {
```

Comment:
We could delete this unless there's a reason it needs to stay commented out

Reply:
Ah thanks for the nudge. All of these commented out tests need to be converted into maze tests. I'll make a ticket for that.